src/super_gradients/training/utils/quantization/export.py

```diff
@@ -0,0 +1,36 @@
+import torch
+
+from super_gradients.common.abstractions.abstract_logger import get_logger
+
+logger = get_logger(__name__)
+
+try:
+    from pytorch_quantization import nn as quant_nn
+
+    _imported_pytorch_quantization_failure = None
+except (ImportError, NameError, ModuleNotFoundError) as import_err:
+    logger.warning("Failed to import pytorch_quantization")
+    _imported_pytorch_quantization_failure = import_err
+
+
+def export_quantized_module_to_onnx(model: torch.nn.Module, onnx_filename: str, input_shape: tuple):
+    """
+    Method for exporting onnx after QAT.
+
+    :param model: torch.nn.Module, model to export
+    :param onnx_filename: str, target path for the onnx file,
+    :param input_shape: tuple, input shape (usually BCHW)
+    """
+    if _imported_pytorch_quantization_failure is not None:
+        raise _imported_pytorch_quantization_failure
+
+    model.eval()
+    if hasattr(model, "prep_model_for_conversion"):
+        model.prep_model_for_conversion()
+    quant_nn.TensorQuantizer.use_fb_fake_quant = True
+    # Export ONNX for multiple batch sizes
+    logger.info("Creating ONNX file: " + onnx_filename)
+    dummy_input = torch.randn(input_shape, device=next(model.parameters()).device)
+    torch.onnx.export(model, dummy_input, onnx_filename, verbose=False, opset_version=13, enable_onnx_checker=False, do_constant_folding=True)
+    # Restore functions of quant_nn back as expected
+    quant_nn.TensorQuantizer.use_fb_fake_quant = False
```

Comment:
Save the  quant_nn.TensorQuantizer.use_fb_fake_quant in the start of the ruction and modify it back to its initial value (of calling this method).